infrastructure/main.tf

```diff
@@ -61,4 +61,10 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   name          = join("-", [var.component, "POSTGRES-DATABASE"])
   value         = module.db-common-data-v11.postgresql_database
   key_vault_id  = data.azurerm_key_vault.rd_key_vault.id
-}
\ No newline at end of file
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_PORT" {
+  name          = join("-", [var.component, "POSTGRES-PORT"])
+  value         = "5432"
+  key_vault_id  = data.azurerm_key_vault.rd_key_vault.id
+}
```

Comment:
@SujeethKumarAri port is not a secret. Can you not hardcode it? It is hardcoded anyway.

Reply:
This is necessary as DB migration method requires PORT to be a secret.